packages/portal-proto/src/features/files/AnnotationsTable.tsx

```diff
@@ -18,11 +18,11 @@ import useStandardPagination from "@/hooks/useStandardPagination";
 import { downloadTSV } from "@/components/Table/utils";
 
 interface AnnotationsTableProps {
-  readonly annotations: ReadonlyArray<FileAnnontationsType>;
+  readonly annotations: ReadonlyArray<FileAnnotationsType>;
 }
-
+// TODO when DEV-2653 is fixed, re-add case ID col, hide case UUID col by default
 type AnnotationTableData = Pick<
-  FileAnnontationsType,
+  FileAnnotationsType,
   | "annotation_id"
   | "case_id"
   | "case_submitter_id"
```

Comment:
is it OK to leave this in?

Reply:
Yeah, that's fine. This is just defining that case_submitter_id is a possible value for the column ids.